### DISCUSSION TITLE
Difference between AlbumGroup and AlbumType

### DISCUSSION THREAD

**Discussion by monezz** [spotify-web-api-java/spotify-web-api-java#325] [Q&A] [answered]
What is the difference between AlbumGroup and AlbumType?
The only addition seems to be the   APPEARS_ON("appears_on") value in AlbumGroup.
Both fields are present in AlbumSimplified.

```
/**
 * An enumeration of all possible album types.
 */
public enum AlbumGroup {

  ALBUM("album"),
  APPEARS_ON("appears_on"),
  COMPILATION("compilation"),
  SINGLE("single");
```

```
/**
 * An enumeration of all possible album types.
 */
public enum AlbumType {

  ALBUM("album"),
  COMPILATION("compilation"),
  SINGLE("single");
```

**Comment by dargmuesli** [comment, marked as answer]
You are right. Please see the Spotify Web Api documentation for this or ask in their forum for reasons behind those data. For album group their documentation (e.g. [get-track](https://developer.spotify.com/documentation/web-api/reference/#/operations/get-track)) says:

> The field is present when getting an artist's albums. Compare to album_type this field represents relationship between the artist and the album.

**Reply by monezz** [reply]
Ah I see, AlbumSimplified defines a relation between an artist and an album and is not just a simplified version of an Album. The group defines the type of relation between the album and the track and the type defines the type of the album itself. Looks a bit more like a graph model opposed to a relational model.

Thanks for the quick reply and the documentation reference, much appreciated!

Edit: You even updated the javadoc of AlbumGroup, sweet!
